public/static/docs/command-reference/update.md

```diff
@@ -69,3 +81,41 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
```

Comment:
```suggestion
## Example: Updating fixed-revision imports to a different version
```
And explain what fixed revision is in the first p. Again, maybe just copy the example from `dvc import`?